### ISSUE TITLE
Support macOS in `kickstart.sh`

### ISSUE THREAD

**Issue by Ferroin** [netdata/netdata#8285]
##### Feature idea summary

Currently, the `kickstart.sh` script does not work correctly on macOS.

While we encourage our users to use Homebrew there to install Netdata, we should still make sure our generic install mechanism works correctly.

##### Expected behavior

You can install Netdata on macOS using `kickstart.sh` just like on most other systems.

**Comment by prologic** [comment]
@Ferroin This is technically in progress right with #8286 ?

**Comment by Ferroin** [comment]
I'd say it's more logically blocked by #8286. We can't have `kickstart.sh` support working without that, but it's not really the kickstart scripts that it's fixing.

**Comment by cakrit** [comment]
TODO: Add documentation

**Comment by prologic** [comment]
@Ferroin Since you are working a fair bit on documentation of late, can you please prepare a draft PR for this and get @joelhans 's help to get it through. I'll ensure we have CI for macOS / FreeBSD in parallel.

**Comment by krital** [comment]
What is the current status of this? I have found some additional issues with kickstart script and wondering if they should be added here or to a new issue

**Comment by Ferroin** [comment]
At this point, I think this should probably be closed (I actually thought we had gotten it working) and then have separate issues opened for the specific problems.

**Comment by krital** [comment]
Ok I have created another issue for my new issues and from my point of view happy to close this one. The new issue is https://github.com/netdata/netdata/issues/10422
